README.md

```diff
@@ -3,28 +3,29 @@
 
 [![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-cosmosdb.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-cosmosdb) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-cosmosdb.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-cosmosdb) [![Build Status](https://dev.azure.com/ms-azuretools/AzCode/_apis/build/status/vscode-cosmosdb)](https://dev.azure.com/ms-azuretools/AzCode/_build/latest?definitionId=7)
 
-Browse and query your MongoDB databases both locally and in the cloud using [_scrapbooks_](#mongo-scrapbooks) with rich Intellisense then connect to Azure to manage your Cosmos DB databases with support for MongoDB, Graph (Gremlin), and SQL (previously known as DocumentDB).
+Browse and query your Azure databases both locally and in the cloud using [_scrapbooks_](#mongo-scrapbooks) with rich Intellisense then connect to Azure to manage your PostgreSQL and Cosmos DB databases with support for MongoDB, Graph (Gremlin), and SQL (previously known as DocumentDB).
 
-![Cosmos DB Extension](resources/features.png)
+![Azure Databases Extension](resources/features.png)
 
 # Prerequisites
 
 - Some less-common commands in the Mongo [scrapbook](#mongo-scrapbooks) and use of the Mongo shell require installing [Mongo DB and Mongo shell](https://docs.mongodb.com/manual/installation/).
+- Executing graph queries for Gremlin accounts requires installing the [Cosmos DB Graph Extension](https://github.com/microsoft/vscode-cosmosdbgraph).
 
 # Features
 
-## Cosmos DB Explorer
+## Azure Databases Explorer
 
-- Create a Cosmos DB account by clicking the `+` button in the title
-- View Azure Cosmos DB accounts and open directly in the portal
-- View/Create/Delete databases, collections, graphs, stored procedures, and documents
-- Click on a document or stored procedure to open in the editor
+- Create a PostgreSQL server or Cosmos DB account by clicking the `+` button in the title

```

Comment:
How about "Create a database server" instead of listing both PostgreSQL and Cosmos DB. Would be consistent with our UI in the extension itself.